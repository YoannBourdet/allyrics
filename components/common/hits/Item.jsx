import React from 'react';
import PropTypes from 'prop-types';

const Item = props => (
  <div style={{ display: 'flex' }}>
    <img src={props.picture} alt="" />
    <p>{props.title}</p>
  </div>
);

Item.propTypes = {
  title: PropTypes.string,
  picture: PropTypes.string,
};

export default Item;
