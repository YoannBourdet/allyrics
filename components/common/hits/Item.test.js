import renderer from 'react-test-renderer';
import React from 'react';
import Item from './Item';

test('ArticleTitleTest renders correctly', () => {
  const tree = renderer.create(<Item title="A title" picture="http://example.com" />).toJSON();
  expect(tree).toMatchSnapshot();
});
