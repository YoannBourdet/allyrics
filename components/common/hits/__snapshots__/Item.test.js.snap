// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleTitleTest renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
    }
  }
>
  <img
    alt=""
    src="http://example.com"
  />
  <p>
    A title
  </p>
</div>
`;
